workshop/exercise-6/README.md

```diff
@@ -277,28 +294,26 @@ git tag 0.2.1-custom -m "Custom collections, version 0.2.1-custom"
 git push --tags
 ```
 
-> FIXME (henrynash): In the following pictures, the release should be 0.2.1-custom, not 0.3.0-custom.
-
 Navigating back to your GitHub repo, you should see a new release available:
 
-![Our own collection, version 0.2.1-custom](images/new-release.png)
+![Our own collection, version 0.2.1-custom](images/new-release-0.2.1-custom.png)
 
 Clicking on the release name (0.2.1-custom) will allow you to edit the release.
 
-![Our own collection, page for version 0.2.1-custom](images/new-release-main.png)
+![Our own collection, page for version 0.2.1-custom](images/new-release-main-0.2.1-custom.png)
 
 Click on *Edit tag*. and then upload all the files in `collections/ci/release/` which were generated from the previous steps, by clicking on the *Attach binaries...* box.
 
 ![Our own collection, upload files to release](images/edit-release.png)
 
 Once you have uploaded the files you can publish your new collection by clicking *Publish release*, at the bottom of the page
 
-![Our own collection, published](images/new-release-published.png)
+![Our own collection, published](images/new-release-published-0.2.1-custom.png)
```

Comment:
since you've added three new images (without a rename) you can delete the old ones